app/insomnia/requests.md

```diff
@@ -0,0 +1,87 @@
+
+tags:
+- requests
+- responses
+- collections
+
+related_resources:
+  - text: Collections
+    url: /insomnia/collections/
+
+faqs:
+- q: What type of requests can I send with Insomnia?
+  a: |
+    Insomnia supports:
+    * HTTP
+    * Event streams (SSE)
+    * GraphQL
+    * gRPC
+    * WebSocket
+---
+
+## How do I create requests in Insomnia?
+
+In Insomnia, requests are contained in collections. Before you can create a request, you need to create a collection. To do that, click **Create** > **Request collection** in your Insomnia project.
+
+Once your collection is created, there are several ways to create requests. Click the `+` button and select an option:
+
+|Option|Steps|
+|---|---|
+|Create a request from scratch|Select the type or request to create (HTTP, Event Stream, GraphQL, gRPC, or WebSocket).|
+|Import a cURL command|Select **From Curl**, then paste your command and click Import.|
+|Import from a file (Postman collection, Swagger, OpenAPI, HAR, WSDL)|Click **From File**, select an import option (file, URL, or clipboard), specify the file to import, then click **Scan** and **Import**.|
+
+## How can I configure requests?
+
+In Insomnia, you can configure every part of your request, and you can also add scripts and documentation.
+
+{:.note}
+> All of these options apply to HTTP, GraphQL, and event stream requests. WebSocket requests have all of these expect for methods and scripts, and gRPC requests have a completely different configuration. <!-- Link to gRPC docs? -->
+
+|Element|Steps|
+|---|---|
+|Method|Select an HTTP method in the dropdown list, or add a custom method.|
+|Endpoint|Enter a URL, and use environment variables and template tags if needed.|
+|Query parameters|In the **Params** tab, click **Add** to create a new query parameter and enter the name and value. You can also add a description. The value can be single line or multiline text. You can use the checkbox next to the parameter to remove it from the URL without deleting it completely.|
+|Body|In the **Body** tab, select the type of body from the dropdown list.|
+|Authentication|In  the **Auth** tab, select the authentication type from the dropdown list to get the corresponding form. If the request is in a folder, you can select **Inherit from parent** to use the folder's authentication.|
+|Headers|In the **Headers** tab, click **Add** to create a new header and enter the name and value. You can also add a description. You can use the checkbox next to the header to remove it from the request without deleting it completely.|
+|Scripts|In the **Scripts** tab, select **Pre-request** or **After-response** and write your script.|
+|Docs|In the **Docs** tab, select **Write** to add documentation to the request. You can use Markdown and HTML syntax, and you can click **Preview** to see the rendered content.|
+
+## What can I do with requests?
+
+You can simply click **Send** to send a request, but you can also click the **Send** menu to see more options.
```

Comment:
Clicking **Send** issues a request to the API. You can access more options by clicking the context menu.